src/components/Plants/PlantingSiteDashboardMap.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { Box } from '@mui/material';
+import { PlantingSite } from 'src/api/types/tracking';
+import { getPlantingSite } from 'src/api/tracking/tracking';
+import useSnackbar from 'src/utils/useSnackbar';
+import { PlantingSitesPlots } from './PlantingSiteDetails';
+import { PlantingSiteMap } from 'src/components/Map';
+
+type PlantingSiteDashboardMapProps = {
+  siteId?: number;
+  plots?: PlantingSitesPlots[];
```

Comment:
Looks like we don't need to pass the plots

Reply:
we need this for the tooltip which is a followup